File: java_backup/my java/ronak/sum2.java
import java.io.*;
class sum2
{
public static void main(String args[])throws IOException
{
int i,n,f;
BufferedReader br=new BufferedReader(new InputStreamReader(System.in));
System.out.print("Give no.");
n=Integer.parseInt(br.readLine());
f=1;
for(i=1;i<=n;i++)
f=f*i;
System.out.print(f);
}
}

File: 6th_sem/MATLAB/Assignment1/sum2.m
%function [s] = sum2(n): Function to calculate s = 1^2 + (1^2+2^2) +
%(1^2+2^2+3^2)+...+(1^2+2^2+...+n^2)

function s = sum2(n)
s2 = 0;
s = 0;
    for i = 1 : n
        s2=s2+i^2;
        s=s+s2;
    end
end
